### PR TITLE
Improvised solution

### PR DESCRIPTION
**Pull request by bconfortin**
This is a work in progress

ForkingBanner's wrapper was modified to <header> in order to avoid conflicts with `.app-styles_Main__content > section` and `<div>` styling in some pages.

A fix using `!important` was considered but discarded.

### CODE REVIEW

File: packages/augur-ui/src/modules/markets-list/components/markets-view.tsx [hunk 1]
```diff
@@ -230,117 +230,119 @@ export default class MarketsView extends Component<
...
-          selectedCategory={selectedCategories}
-          search={search}
-          updateMobileMenuState={updateMobileMenuState}
-        />
-
-        <div className={classNames({
-          [Styles.Disabled]: isSearching,
-        })}>
-          <MarketTypeFilter
+      <>
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/markets-list/components/markets-view.tsx]: Will remove this fragment.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💫

 This link will always point to this PR's latest changes: https://peek.run/exoh0qj

**Comment by bthaile** [comment]
Chatted with @phoebemirman about the banner. She suggested putting the banner in App.tsx. It does make logical sense but I think there is an issue with the responsiveness stuff pushing the banner under the header and navigation. Is that what you were seeing?

**Comment by bconfortin** [comment]
> Chatted with @phoebemirman about the banner. She suggested putting the banner in App.tsx. It does make logical sense but I think there is an issue with the responsiveness stuff pushing the banner under the header and navigation. Is that what you were seeing?

@bthaile I'm testing it right now. First attempt: it breaks the scrolls.

**Pull request closed without merging**